src/Command/LoadScreenLayoutsCommand.php

```diff
@@ -37,7 +38,23 @@ final protected function execute(InputInterface $input, OutputInterface $output)
             try {
                 $content = json_decode(file_get_contents($filename), false, 512, JSON_THROW_ON_ERROR);
 
-                $screenLayout = new ScreenLayout();
+                if (isset($content->id)) {
```

Comment:
You should either add `&& Ulid::isValid($content->id)` to the `if`clause or catch the `\InvalidArgumentException` thrown by `Ulid::fromString`on inavlid value